mlflow/pytorch/__init__.py

```diff
@@ -32,11 +32,13 @@
 from mlflow.utils.environment import _mlflow_conda_env
 from mlflow.utils.file_utils import _copy_file_or_tree, TempDir
 from mlflow.utils.model_utils import _get_flavor_configuration
+from mlflow.utils.annotations import experimental  # pylint: disable=reimported
```

Comment:
Why do we need `# pylint: disable=reimported` here?

Reply:
without this disable, pylint is throwing a reimported error (eventhough experimental is not imported before)

Reply:
Can you merge the master branch and check if the same error occurs?

Reply:
sure will do

Reply:
merged master branch and the issue got fixed.